packages/shared/src/storage/GoogleDriveStorage/GoogleDriveStorage.ts

```
@@ -0,0 +1,357 @@
+      throw new OneKeyLocalError(
+        'Google Drive is only supported on Android currently. Desktop and Web support coming soon.',
+      );
+    }
+  }
+
+  /**
+   * Generate temporary file path for Google Drive operations
+   */
+  private getTempFilePath(fileName: string): string {
+    if (!RNFS) {
+      throw new OneKeyLocalError('File system not available');
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)